src/js/rx/components/Breadcrumbs.jsx

```diff
@@ -30,7 +30,7 @@ class Breadcrumbs extends React.Component {
     }
 
     return (<div className="rx-breadcrumbs">
-      {crumbs.reduce((content, e) => { return [...content, ' › ', e]; }, crumbs[0]).slice(1)}
+      {crumbs.reduce((content, e) => { return [...content, ' › ', e]; }, []).slice(1)}
```

Comment:
that causes the first element to appear twice; the following code works but I prefer to use `slice` to avoid the ternary. `slice` is necessary to avoid the hanging `›` in front of the first element

`crumbs.reduce((content, e) => { return content.length > 0 ? [...content, ' › ', e] : [e]; }, [])`